qa/automation/CareerPortal.Tests.Scenarios/1.Send your CV/Send CV.feature

```diff
@@ -0,0 +1,275 @@
+	| "Message" | "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum.idunt ut labore et dolore magna aliqua.Lorem ipsumshtrsrthtrrshsrthhsrthtrsthsrthsrhssrththsrhsrthtrhshtrhg dvaebaere" | 500    |
+	
+
+# отличная от класса эквивалентности проверка
+Scenario Outline: 1.1.11. Validation of copied data in text fields
+	When user copies valid <Field> from any document with meaning <Description>
+	And user puts this <Description> in <Field> text field
+	Then user sees that the line under the text field is highlighted in green
+
+	Examples: 
+	| Field     | Description |
+	| "Name"    | O O         |
+	| "Email"   | f@e.ru      |
+	| "Message" | 1213        |
+	
+	
+
+@5
+@Placeholders
+
+Scenario Outline: 1.1.12. Checking the placeholders in  text fields
+    Given user sees the plceholder <Placeholder> in first text field 
+	When user clicks in <Text field> text field and fill in with ""
+	And user clicks on other place
+	Then placeholder <Placeholder> appears back
+
+	Examples: 
+	| Placeholder | Text field |
+	| Name*       | Name       |
+	| Email*      | Email      |
+	| Message     | Message    |
+
+Scenario Outline: 1.1.13. Checking the disappearing of placeholders in  text fields
+    Given user sees the plceholder <Placeholder> in first text field 
+	When user clicks in <Text field> text field and fill in with " "
+	And user clicks on other place
+	Then placeholder <Placeholder> disappears
+
+	Examples: 
+	| Placeholder | Text field |
+	| Name*       | Name       |
+	| Email*      | Email      |
+	| Message     | Message    |
+
+@6
+@AttachCVbutton
+
+Scenario Outline: 1.1.14. Attaching a valid document 
+	When user clicks on "Attach CV" button
+	And user attaches a file in <format> format weighing less or equal 10 Mb with the name <name>
+    Then user sees that under "Attach CV" button attached document <name> appeared 
+
+	Examples: 
+	| format | name      |
+	| PDF    | name.pdf  |
+	| DOC    | name.doc  |
+	| DOCX   | name.docx |
+	| RPF    | name.rpf  |
+
+
+
+Scenario Outline: 1.1.15. Attaching invalid document 
+    When user clicks on "Attach CV" button
+   	And user attaches a file in <format> format weighing less or equal 10 Mb with the name <name>
+    Then user sees a warning message "Something went wrong Invalid file format. Please try PDF, DOC, DOCX or RTF." and "OK" button
+	And user clicks on "OK" button
+	And user sees that <name> did not attach
+
```

Comment:
I guess, to avoid multiple "When" steps, a "Given" option should be introduced.

Like : 
Given user clicked on "Attach CV" button
And user attached a file in (format) format weighing less or equal 10 Mb with the name (name)
When a warning message "Something went wrong Invalid file format. Please try PDF, DOC, DOCX or RTF." is displayed
Then (name) file is not attached